.cirrus.yml

```diff
@@ -0,0 +1,64 @@
+common_steps_template: &COMMON_STEPS_TEMPLATE
+  clone_dmd_script: |
+    set -uexo pipefail
+    git clone --branch "${CIRRUS_BASE_BRANCH:-$CIRRUS_BRANCH}" --depth 1 https://github.com/dlang/dmd.git ../dmd
+  install_prerequisites_script: cd ../dmd && ./cirrusci.sh
+  install_host_compiler_script: cd ../dmd && ./ci.sh install_host_compiler
+  setup_repos_script: |
+    set -uexo pipefail
+    ln -s $CIRRUS_WORKING_DIR ../phobos
```

Comment:
Unblocked & ready. This file is a copy of https://github.com/dlang/druntime/pull/3235, the only change being the `../phobos` symlink in the line above.